### DISCUSSION TITLE
Digest Auth

### DISCUSSION THREAD

**Discussion by mobile-tramontina** [viniciussanchez/RESTRequest4Delphi#94] [General]
Olá galera, alguém sabe se tem como autenticar usando o tipo Digest Auth pelo RESTRequest4Delphi?

![postman](https://user-images.githubusercontent.com/73296658/170480746-84e7de53-e319-4e6e-8f5e-e68bf2b18e19.jpg)

**Comment by viniciussanchez** [comment]
Normalmente ele adiciona o Header Authorization.
![image](https://user-images.githubusercontent.com/16382981/170486988-6e9b776a-22a2-456b-81d1-de50cf84b0db.png)

**Reply by mobile-tramontina** [reply]
Olá Vinicius

Apenas para dar um feedback, infelizmente não consegui fazer a autenticação authDigest com o RESTRequest4Delphi, tive que usar o componente TipwHTTP. Se alguém souber como fazer com o RESTRequest4Delphi, ficaria grato com a dica, pois a API é sensacional. 

object ipwHTTPAcesso: TipwHTTP
  AuthScheme = authDigest
  Password = 'my_password'
  User = 'my_user'
  OnTransfer = ipwHTTPAcessoTransfer
end

**Reply by sbnn** [reply]
Boa tarde.

Estou tentando fazer autenticação digest no IdHttp, mas sem sucesso até agora.
Me fale desse componete "TipwHTTP". onde comprar.

**Reply by ZeCarlinhos** [reply]
https://www.nsoftware.com/ipworks

**Comment by ZeCarlinhos** [comment]
Vinicius, estou com o mesmo problema. como utilizar o Auth Digest?
